app/views/shared/_sequence_next_prev.html.haml

```diff
@@ -3,11 +3,11 @@
 .intra-sequence-nav
   .next-activity{ :class => ('disabled' if next_activity.nil?) }
     - if next_href
-      = link_to 'Next Activity >', sequence_activity_path(sequence, next_activity), { :title => next_activity.name }
+      = link_to 'Next Activity >', runnable_activity_path(sequence, next_activity), { :title => next_activity.name }
```

Comment:
This looks broken. `runnable_activity_path` doesn't seem to take a sequence as a param.